content/docs/command-reference/commit.md

```diff
@@ -279,5 +279,5 @@ $ dvc status
 Data and pipelines are up to date.
 ```
 
-Nothing special is required, we simply `commit` to both the SCM and DVC. Since
+Optimization of resources can be easily maintained. Use `commit` to both Git and DVC, when ready. Since
 this pipeline is up to date, `dvc repro` will not do anything.
```

Comment:
Hmmm... I'm not sure what this new sentence means either. Please try to understand the context and close the idea here. It's something along the lines of being able to force-register the changes to a stage/pipeline without reproducing it. It's best if you actually try the example to fully get the idea 🙂

Reply:
May also want to try basic dvc run and dvc repro. See the examples in their cmd refs. Thanks

Reply:
Okay.

Reply:
Moving to https://github.com/iterative/dvc.org/pull/1327#pullrequestreview-421506486